src/domain/Callouts/EmptyState/EmptyState.tsx

```diff
@@ -20,6 +21,8 @@ export interface IEmptyStateProps {
   margins?: Props.IMargins
   /** The data-component-context */
   componentContext?: string
+  /** Background color of the component */
+  backgroundColor?: Variables.Color
```

Comment:
I think the card was to make this a `isTransparentBackground` card, rather than being a configuration of colour